IdentityCore/src/webview/systemWebview/MSIDSFAuthenticationSession.m

```diff
@@ -94,12 +104,22 @@ - (void)startWithCompletionHandler:(MSIDWebUICompletionHandler)completionHandler
                                                 }
 
                                                 [[MSIDTelemetry sharedInstance] stopEvent:_telemetryRequestId event:_telemetryEvent];
-                                                completionHandler(callbackURL, error);
+                                                
+                                                [self notifyEndWebAuthWithURL:callbackURL error:error];
+                                                _completionHandler(callbackURL, error);
                                             }];
-        if ([_authSession start]) return;
+        
+        [MSIDNotifications notifyWebAuthDidStartLoad:_startURL];
+        if (_authSession && [_authSession start]) return;
+        
+        error = MSIDCreateError(MSIDErrorDomain, MSIDErrorInteractiveSessionStartFailure, @"Failed to start an interactive session", nil, nil, nil, _context.correlationId, nil);
```

Comment:
nit: would be easier to read if we check for (![_authSession start]) and then create an error.

Reply:
It would cause else block to return which looks weird.
But looking back, there is no need to check for _authSession as if it is nil, [_authSession start] will return NO anyways.